src/main/java/config/Config.java

```diff
@@ -0,0 +1,72 @@
+package config;
+
+import executor.Executor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import task.SubTask;
+import task.Task;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class Config {
+
+    @Bean
+    public Task task1() {
+        Task task1 = new Task("№ 1");
+        SubTask subTask1 = new SubTask("первой подзадача", task1);
+        SubTask subTask2 = new SubTask("вторай подзадача", task1);
+        return new Task("№ 1") {
+            @Override
+            public void execute() {
+                super.execute();
+                subTask1.execute();
```

Comment:
We shouldn't implement it such way!
This override here is not correct and it should be placed in the Yask.execute method.
You can just store List of SubTasks, then iterate over them and call method